.circleci/config.yml

```diff
@@ -185,11 +185,39 @@ jobs:
             errors=$(jq '.["results"] | .[] | unique_by(.["type"]) | map(select(.["type"] == "error")) | .[]' planet4-docker-compose/artifacts/pa11y/pa11y-ci-results.json)
             if [ ! -z "${errors}" ]; then echo "Errors found, see report in artifacts." && exit 1; else echo "No errors, report available in artifacts."; fi
 
-  rips-test:
+  create-release-zip:
     docker:
-      - image: rips/rips-cli:3
-    environment:
-      RIPS_APPLICATION_ID: 21
+      - image: mhart/alpine-node
     steps:
+      - run: apk add curl zip git openssh-client jq python make g++
       - checkout
-      - run: rips-cli -vvv rips:scan:start -a $RIPS_APPLICATION_ID -p ~/project/ -t 1
+      - run: git submodule init && git submodule update
+      - run: PUPPETEER_SKIP_DOWNLOAD=true PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true npm ci
+      - run: npm run build
+      - run:
+          name: "Remove files we don't want in the zip file."
+          command: rm -rf .circleci .git .githooks assets/src bin tests
```

Comment:
I included the ones of which I was sure they wouldn't be needed in the zip. Maybe there's some others which would make sense to exclude here? The total zip size is currently about 1.8MB.